commands/daily.py

```diff
@@ -15,6 +15,13 @@ def __init__(self, client):
 
     @commands.command()
     async def daily(self, ctx):
+        
+        def _update_advent():

```

Comment:
According to PEP 8, somewhat private functions should be prefixed with an `_`, given that we're importing daily (as this is a cog) it seemed wise to do so. (ref: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)

Also `update_lastClaimed_and_streaks` is a mix of `snake_case` and `camelCase`, but I'm open to changing it to something else.

Reply:
the reason I suggested that is because we're updating the lastClaimed and streaks values within the method, if you want to follow the naming convention for protected methods and variables go for it. Please do this asap as I would like to have this fix in 3.6.1 which is releasing in a few hours.

Reply:
Let's create an issue for 3.6.2 regarding that, I'll change that to your suggested and create a separate PR for the naming conventions

Reply:
I wasn't planning on a 3.6.2 release, 3.7.0 is the next release (in January), if you're not able to do it for 3.6.1 then I will make the relevant adjustments myself in this PR.

Reply:
I mean, how long do I have?

Reply:
Until Midnight :P